osf/models/user.py

```diff
@@ -1715,7 +1769,7 @@ def gdpr_delete(self):
         )
         shared_nodes = user_nodes.exclude(id__in=personal_nodes.values_list('id'))
 
-        for node in shared_nodes.exclude(type='osf.quickfilesnode'):
+        for node in shared_nodes.exclude(type__in=['osf.quickfilesnode', 'osf.draftnode']):
```

Comment:
hmm, this probably isn't right. Need to think more about GDPR delete.  Handle similarly to preprints?  What about draft registrations? Do we care about those?

Reply:
Reticketed